app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_refer_case.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+    @case_heading(c)
+
+    <div class="grid-row">
+        <div class="column-full">
+            <h3 class="heading-large mt-0">This case has been referred</h3>
+
+            <p>The days elapsed count for this case has been paused until it is reopened.</p>
+            <p>The status of this case is now referred.</p>
+            <p> Work on another case from
```

Comment:
On other confirmation (e.g. Release, Complete Case) pages we have a different "Next Steps" style.
I don't mind which we use but we should be consistent across them all

Reply:
I agree, now they are slightly different because the prototypes... There will be an story to align this prototypes... not sure what will be the last design..,

Reply:
Just in case that story never happens though I'd keep them all consistent